content/master/cli/command-reference.md

```diff
@@ -504,13 +496,13 @@ already installed in Crossplane.
 
 `crossplane xpkg update <package-kind> <registry package name and tag> [<optional-name>]`
 
-The package file must be an organization, image and tag on the `xpkg.upbound.io`
-registry on [Upbound Marketplace](https://marketplace.upbound.io/).
+The package file must be an organization, image and tag on the `xpkg.crossplane.io`
+GitHub Container Registry in [crossplane-contrib](https://github.com/orgs/crossplane-contrib/packages) GitHub org.
```

Comment:
looking at the source code implementation, I don't think this statement has ever been true - I think the code would always have worked for any registry. Maybe let's just remove this paragraph completely 🤪